iot-hub/Tutorials/Routing/SimulatedDevice/resources/iothub_msgenrichment_cli.azcli

```diff
@@ -83,10 +83,10 @@ az iot hub device-identity show --device-id $iotDeviceName \
 ##### ROUTING FOR STORAGE #####
 
 # You're going to have two routes and two endpoints.
-# One points to container1 in the storage account
-#   and includes all messages.
+# One route points to container1 in the storage account
```

Comment:
Is it really called 'container1' ?

Reply:
It's not - I think someone included this as shorthand to tell them apart. I'll put in the actual container name.